src/types.ts

```diff
@@ -40,8 +40,11 @@ export type SwipeableCallbacks = {
 };
 
 // Configuration Options
+export type ConfigurationOptionDelta =
+  | number
+  | { [key in Uncapitalize<SwipeDirections>]: number };
```

Comment:
💭 🤔 going to take a look at making each direction optional and defaulting to the defaultProps.delta value if not present